sql3/test/defs/defs_groupby.go

```diff
@@ -257,3 +257,171 @@ var groupByTests = TableTest{
 		},
 	},
 }
+
+// groupby/distinct with sets tests
+var groupBySetDistinctTests = TableTest{
+	Table: tbl(
+		"groupby_set_test",
+		srcHdrs(
+			srcHdr("_id", fldTypeID),
+			srcHdr("ids1", fldTypeIDSet),
+			srcHdr("ss1", fldTypeStringSet),
+		),
+		srcRows(
+			srcRow(int64(1), []int64{1, 2}, []string{"a", "b"}),
```

Comment:
I'd like to have at least one duplicate set value so the counts we're testing against aren't all 1, which makes it easier to verify that the flattening, etc., is working correctly.